### ISSUE TITLE
Upgrade to Laravel 7

### ISSUE THREAD

**Issue by bryanjamesmiller** [optimistdigital/nova-sortable#26]
Hi, thanks for your nice package!

Can you make a release that will work on Laravel 7?  At least one of your dependencies is preventing this: https://github.com/spatie/eloquent-sortable/blob/3.7.0/composer.json - thanks!

**Comment by Tarpsvo** [comment]
Hmm, I have the rule set up as `"spatie/eloquent-sortable": "^3.7.0",` which should also include `3.8.0` which has 7.0 support. You sure my package is blocking it? It installed 3.8.0 for me in my test project.

**Comment by bryanjamesmiller** [comment]
@Tarpsvo Hi, Thank you for the quick reply.  I was able to get it to install after doing the following:  

1. I deleted your package and another one that was blocking our upgrade in addition to yours (apparently) and upgrading everything else to Laravel 7 and Nova 3

2. I re-added your package via `composer require optimistdigital/nova-sortable`.  It successfully added.  

So I'm not sure why the earlier workflow was causing issues, but it looks like your package is compatible with Laravel 7 after all.  Thanks again!
